README.md

```diff
@@ -49,6 +55,9 @@ If you've used the shell script or if you've checked out, installed and built al
   ```
 - restart `next-router`
 - `cd` into `next-syndication-api` and `make run-local`
+- go to https://local.ft.com:5050/syndication/user-status to confirm everything is working
+
+If you see an error that port 3255 is already being used, use `lsof -n -i :3255 | grep LISTEN` to find out its PID and `ps -p THE_PID` to find out what that process is. Sometimes PM2 doesn't stop properly, and you can use `kill THE_PID` to stop it.
```

Comment:
I wonder if we need this step? Is this going to be a common thing?

Reply:
the 3255 port error? maybe not, that was because I'd been running make run-local and make run.

Reply:
Maybe see what Alice comes up against?